test/TestAsset/NullDefaultHintsClass.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
```

Comment:
`EmptyClass` includes the license docblock, but this and those following do not; should they? or are they used for comparison purposes?